src/sentry/migrations/0584_backfill_alert_rule_user_and_team.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.2.28 on 2023-04-26 19:54
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_alert_rule_user_and_team(apps, schema_editor):
+    AlertRule = apps.get_model("sentry", "AlertRule")
+    for ar in RangeQuerySetWrapperWithProgressBar(
+        AlertRule.objects.filter(owner_id__isnull=False).select_related("owner")
+    ):
+        ar.update(team_id=ar.owner.team_id, user_id=ar.owner.user_id)
```

Comment:
`.update` doesn't exist in migrations. You can set the attributes on the model, then pass a list of fields to the .save() method